base_suspend_security/models/ir_model_access.py

```diff
@@ -17,19 +17,18 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from openerp import models, tools
+from odoo import models, api, tools
 from ..base_suspend_security import BaseSuspendSecurityUid
 
 
 class IrModelAccess(models.Model):
     _inherit = 'ir.model.access'
 
-    @tools.ormcache_context('uid', 'model', 'mode', 'raise_exception',
+    @api.model
+    @tools.ormcache_context('self._uid', 'model', 'mode', 'raise_exception',
                             keys=('lang',))
```

Comment:
@adrienpeiffer that's a good reason, but not a good explanation ;-) - I believe https://github.com/OCA/OCB/blob/10.0/odoo/addons/base/ir/ir_model.py#L828-L831 phrases it sufficiently. Given it's still only the `returns` decorator which is propagated in inheritance (https://github.com/OCA/OCB/blob/10.0/odoo/api.py#L99, https://github.com/OCA/OCB/blob/10.0/odoo/api.py#L127, https://github.com/OCA/OCB/blob/10.0/odoo/api.py#L77), I think it's the right thing to do to copy the original, even though it doesn't have an effect currently save for being slightly less efficient in cases of switching language